tests/common/file_utils.rs

```diff
@@ -15,9 +18,18 @@ use std::path::PathBuf;
 /// get_path_in_temp(&path_in_temp);
 ///
 pub fn get_path_in_temp(file_path: &str) -> PathBuf {
-    let temp_dir = Temp::new_dir().unwrap();
+    let temp_dir = mktemp::Temp::new_dir().unwrap();
     let mut path = temp_dir.to_path_buf();
     path.push(&file_path);
     temp_dir.release();
     path
 }
+
+/// Creates file in temporary folder
```

Comment:
The docs should decide if they want to use "folder" or "directory". I'd prefer the latter, "folder" is a UI term :)